src/e2e-selectors/types.ts

```diff
@@ -0,0 +1,682 @@
+import { versionedComponents, versionedPages } from './versioned';
+import { versionedAPIs } from './versioned/apis';
+
+export type E2ESelectors = {
+  pages: Pages;
+  components: Components;
+  apis: APIs;
+};
+
+export type VersionedSelectors = {
+  pages: typeof versionedPages;
+  components: typeof versionedComponents;
+  apis: typeof versionedAPIs;
+};
+
+export type APIs = {
+  DataSource: {
+    getResource: string;
+  };
+};
+
+export type Components = {
```

Comment:
If possible, it would be nice to scrap these types and add new selectors step by step as we work with this library. So basically that we just start to support the basic scenarios to begin with.

Reply:
Yeah I agree. Meanwhile developing, it's kind of convenient to have all of them so you can search and find easily, but when we're ready for a first public release this should definitely be removed.